### DISCUSSION TITLE
Solve problem on Windows 11 Insider 25941 showing Black Screen

### DISCUSSION THREAD

**Discussion by aldeniangman** [valinet/ExplorerPatcher#1973] [Show and tell]
If you see a black screen after updating to Windows 11 Insider Build 25941. Here is what you need to do:

1. Press Ctrl+Alt+Delete.
2. Hold shift while clicking Power Button > Restart. This will take you to Recovery Mode
3. Click Troubleshoot.
4. Click Advanced options.
5. Click Startup Options.
6. Click Restart.
7. If you have BitLocker on, enter your Bitlocker key.
8. Click 1 on your keyboard or F1 on your keyboard.
9. It will Restart.

This will fix the problem, not the taskbar problem of not applying settings in ExplorerPatcher.

**Comment by aldeniangman** [comment]
Please note after rebooting, it will temporarily disable the debugging mode and after you do that again Chrome will wipe all data except Google Data.

**Comment by pyrates999** [comment]
Insider builds are not supported by EP.

**Reply by aldeniangman** [reply]
😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡stop saying that😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡you made me mad😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡dont say that again😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡LISTEN?😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡

**Comment by aldeniangman** [comment]
also after solving the problem and back to normal. you will lose your new chrome update and back to old chrome version and cannot update it to new version.

**Reply by aldeniangman** [reply]
oh that was chrome canary

**Reply by aldeniangman** [reply]
because of chrome.exe is chrome canary

**Comment by demcreative** [comment]
Just had to uninstall it, sadly. I dunno if this app was causing sudden system reboot but from what I read the black screen with only the cursor showing up and no way to open task manager or using any other shortkey including the Windows button might be an issue. A message from EP besides said Explorer started and crashed several times after rebooting and logging in, suggesting to uninstall EP. So I did.
The way Windows updates mess things up for my laptop is killing me with concerns about the fact hardware fatal issues kill my laptop again after HP already changed its motherboard a year or less ago. At that time it was still under guarantee but now it's not. When this laptop is done are gonna be really though times for me.
It sucks for me to have the taskbar on the bottom, it really kills my workflow, but better the annoying bottom taskbar than being with no laptop at all. Pity, EP does well what's supposed to o, when Wondows doesn't turn nasty!

**Reply by aldeniangman** [reply]
why won't valinet fix our problems

**Reply by pyrates999** [reply]
insider builds and preview updates are not supported.  So don't expect him to fix the issues you are bringing up if they only are happening on an insider build or with preview updates.

**Reply by aldeniangman** [reply]
I will switch to stable and then when the problems are solved i will go back to insider
